src/beacons/BeaconsContext.tsx

```diff
@@ -135,15 +133,9 @@ const BeaconsContextProvider: React.FC = ({children}) => {
   const deleteCollectedData = useCallback(async () => {
     if (!isBeaconsSupported) return;
     if (isInitializedRef.current) {
-      const deleteOK = await Kettle.deleteCollectedData()
-        .then(() => {
-          return true;
-        })
-        .catch((error) => {
-          Bugsnag.notify(error);
-          return false;
-        });
-      return deleteOK;
+      await Kettle.deleteCollectedData().catch((error) => {
```

Comment:
```suggestion
     Kettle.deleteCollectedData().catch((error) => {
```
Sorry I meant to exclude await from the previous suggestion, so it can be removed I think